details.py

```diff
@@ -1,20 +1,36 @@
-def contact_info(contact_name,contact_phone_number,contact_email,contact_dic ):
-    contact_dic[contact_name] = [contact_phone_number,contact_email]
-    return contact_dic
-cd = dict()
+import os
+import json
 
-name1 = input("Contact Name: ")
-phone_number1  = int(input("Contact Phone NUmber: "))
-email1 = input("Contact Email Address: ")
-print(contact_info(name1,phone_number1,email1,cd))
+option = input(
```

Comment:
Between here and line 7, I don't understand what it's meant to be. Can you explain what it is?

Reply:
@Anitaachu

Reply:
I believe this shouldn't be added in this commit a you're trying to solve #4 in this PR. Each pull request should be solving one particular issue or maybe related issues. So a PR could possibly serve for #4 and #5 and then you do a PR to handle that